### DISCUSSION TITLE
官方文档显示Cora的边数是10,556，但是实际上是5429

### DISCUSSION THREAD

**Discussion by Chrisable** [pyg-team/pytorch_geometric#4069] [General]
![image](https://user-images.githubusercontent.com/49330390/153694776-42efe681-f172-4460-bde6-a6c1d732b89b.png)
官方文档显示Cora的边数是10,556，但是实际上是5429，其他数据集也有类似问题

**Comment by rusty1s** [comment]
What's the issue here? Is it that the number of edges vary from those reported in literature? We are reporting the number of bidirectional edges here. Furthermore, we are removing some duplicated edges that are present in the initial version of the dataset.

**Reply by Chrisable** [reply]
就算除以2也不是5429，应该是还存在重复边，请问什么时候可以删除完重复边呢？

**Reply by rusty1s** [reply]
The number of edges matches prior to removing duplicated ones. You can test that by yourself by removing [these](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/io/planetoid.py#L115-L119) lines. In the end, we simply clean up the dataset, and there does not exist any significant difference in datasets as well as model performances between both versions.

**Reply by Chrisable** [reply]
哪一个函数会调用planetoid.py呢？

**Reply by rusty1s** [reply]
Can you explain what you mean?

**Reply by Chrisable** [reply]
Removing these lines doesn't solve the problem, what should I do?

**Reply by rusty1s** [reply]
Where did you remove these lines? Keep in mind that you might need to re-install if you do any modifications to the source code.

**Comment by Chrisable** [comment]
when i remove these lines
![image](https://user-images.githubusercontent.com/49330390/165278336-af774949-857e-4756-96ba-deccf255985c.png)
there is a  error
![image](https://user-images.githubusercontent.com/49330390/165278456-d198ada5-25c8-4df1-b65d-c9fd259a29b6.png)

**Reply by rusty1s** [reply]
You need to remove these lines: https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/io/planetoid.py#L117-L121

**Comment by Chrisable** [comment]
Thank for your reply.my question is solved.thank you very much ！！！
